content/docs/guides/crossguard/awsguard.md

```diff
@@ -0,0 +1,251 @@
+---
+title: Pulumi CrossGuard policies for AWS (AWSGuard)
+linktitle: AWSGuard
+
+menu:
+  userguides:
+    parent: crossguard
+    weight: 2
+---
+{{% crossguard-preview %}}
+
+## Overview
+
+AWSGuard codifies best practices for AWS. This is a configurable library that you can use to enforce these best practices for your own Pulumi stacks or organization.
```

Comment:
I'd definitely add something about it being an open-source policy pack, with the intention of helping direct external contributors to helping flesh out aspects of it.

Perhaps just add another sentence like "It's [source code is available], allowing you to add new policies to its ever expanding set." or something?